notebooks/5_CUR.ipynb

```diff
@@ -1161,9 +1165,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "lr_pcov = LR(regularization=1e-6)\n",
+    "lr_pcov = LinearRegression()\n",
```

Comment:
Since the first three cases only use LR instead of ridge regression, I also use LR here.
![image](https://user-images.githubusercontent.com/56018055/100129356-c377cf80-2e81-11eb-942d-9ca579652f5a.png)